StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1130UnitTests.cs

```diff
@@ -380,5 +380,118 @@ public static void TestMethod2(TestDelegate testDelegate)
 
             await VerifyCSharpFixAsync(testCode, expected, fixedCode, CancellationToken.None).ConfigureAwait(false);
         }
+
+        [Fact]
+        public async Task TestDelegateUseAsConstructorArgumentsAsync()
```

Comment:
❓ Can we also add tests for references to overloaded indexers?

Reply:
Good catch! This actually caused the code to crash again because BracketedArgumentListSyntax cannot be casted to ArgumentListSyntax.